ebs/ebs.go

```diff
@@ -348,51 +364,36 @@ func (d *Driver) MarkVolumeForGC(volume *ec2.Volume) error {
 	return nil
 }
 
-func (d *Driver) CreateAndBuildFromSnapshot(oldVolume *ec2.Volume, snapshotVolume *ec2.Volume, opts map[string]string) (string, int64, error) {
-	// If the volume I would snapshot off of is the same as the one that I need to create then just return
-	if oldVolume != nil && *oldVolume.VolumeId == *snapshotVolume.VolumeId {
-		return *oldVolume.VolumeId, *oldVolume.Size * GB, nil
-	}
+func (d *Driver) CreateAndBuildFromSnapshot(volume *ec2.Volume, args *BuildArgs) (string, int64, error) {
```

Comment:
yeah, let's keep 3 args. Although we might want to create a convention within medallia for not having naked returns :)